File: 10 - Python/Exercícios/04 - Vetores/Exercício 05.py
#Exercício - Maior Posição

N = int(input("Quantos números você vai digitar? "))

vet = [0] * N

maior = vet[0]
posicao = 0

for i in range(N):

    vet[i] = float(input("Digite um número: "))

    if maior < vet[i]:
        maior = vet[i]
        posicao = i

print("")
print("Maior valor =", maior)
print("Posição do maior valor =", posicao)

File: 06 - C/Exercícios/04 - Vetores/Exercício 05.c
//Exercício - Maior Posição

#include <stdio.h>

int main(){

    int i, N, posicao;
    double maior;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    double vet[N];

    for(i = 0; i < N; i++){
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    maior = vet[0];
    posicao = 0;

    for(i = 0; i < N; i++){
        if(vet[i] > maior){
            maior = vet[i];
            posicao = i;
        }
    }

    printf("\nMaior valor = %1.lf\n", maior);
    printf("Posicao do maior valor = %d", posicao);

    return 0;

}